src/main/java/Duke.java

```diff
@@ -1,10 +1,135 @@
+import Tasks.*;
+import java.io.IOException;
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+    private Storage storage;
+    private ArrayList<Task> todolist = new ArrayList<Task>();
+    private Ui ui;
+
+    /**
+     * Initialize the Duke.
+     * @param filePath
+     */
+    public Duke(String filePath){
+        ui = new Ui();
+        storage = new Storage(filePath);
+        Storage.create(filePath);
+        try {
+            todolist = new ArrayList<Task>(storage.load());
+        } catch (IOException e) {
+            ui.showLoadingError();
+            todolist = new ArrayList<Task>();
+        }
+    }
+
+    public void run() throws IOException {
+        String task;
+        ui.greetings();
+        Scanner in = new Scanner(System.in);
```

Comment:
Maybe you should consider a more descriptive name for the Scanner object like "scanner"?